lib/galaxy/datatypes/genetics.py

```diff
@@ -518,6 +518,73 @@ class Snptest(Rgenetics):
     file_ext = "snptest"
 
 
+class IdeasPre(Rgenetics):
```

Comment:
You can probably use `Html` here instead of `Rgenetics`

Reply:
@nsoranzo https://github.com/galaxyproject/planemo/issues/760

Reply:
Why would that be the case?

Reply:
@nsoranzo The main reason is that the Galaxy upload tool will not allow uploading the Html datatype - https://github.com/galaxyproject/galaxy/blob/dev/tools/data_source/upload.py#L277.  But tool function al tests require the use of the upload tool.

Reply:
@gregvonkuster That was my point, subclassing Rgenetics and overload all its method is equivalent to subclass Html. But I see you changed it already, thanks!